src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
For such repeatable actions it's better to use for loop, just remember to not create objects like services inside the loop:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```
Also do not create class instances with names which may be misunderstood, or lead to some mistakes. When we create an instance of a `Lottery` class it is better to name the variable `lottery` instead of `ball`.
Lastly you don't need to explicitly call `toString()` method as it will be implicitly called by `System.out.println()`.

For the reference please see the following links:

- https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes
- https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class